scripts/functions/selector

```diff
@@ -383,6 +383,7 @@ __rvm_ruby_string_find()
   else
     __rvm_ruby_string_autodetect
     case "${rvm_ruby_string}" in
+      # Should truffleruby-* be added here?
```

Comment:
Should truffleruby-* be added here? It's not clear to me.

Reply:
no need